docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/ImageClassification.cs

```diff
@@ -86,5 +97,31 @@ class OutputScores
         {
             public float[] output { get; set; }
         }
+
+        private static string Download(string baseGitPath, string dataFile)
+        {
+            using (WebClient client = new WebClient())
+            {
+                client.DownloadFile(new Uri($"{baseGitPath}"), dataFile);
+            }
+
+            return dataFile;
+        }
+
+        /// <summary>
+        /// Taken from https://github.com/icsharpcode/SharpZipLib/wiki/GZip-and-Tar-Samples.
```

Comment:
>https://github.com/icsharpcode/SharpZipLib/wiki/GZip-and-Tar-Samp [](start = 23, length = 65)

Don't we have anything in .NET for zip/unzip. Are we fine to use external libraries?

The link is for wiki section. I think it should be for main repo, right?

Reply:
Nope. I tried but this is the most talked library.